src/components/implementation/tests/unit_cbufmalloc/posix_test.c

```diff
@@ -0,0 +1,48 @@
+#include <printc.h>
+#include <cos_component.h>
+#include <cbuf_mgr.h>
+#include <assert.h>
+
+#define NUM 10
+char *mptr[NUM];
+
+void
+cos_init(void *args)
+{
+	int err=0;
+	int i, j;
+
+	for (i = 0; i < NUM; i++) {
+		mptr[i] = (char *)malloc((i+1) * sizeof(int));
+
+		assert(mptr[i]);
+		
+		for (j = 0; j < i; j++) {
```

Comment:
This is small:  `for (j = 0 ; j < i ; j++) {...` is correct.  Notice the use of spaces.